.github/workflows/pkgdown.yaml

```diff
@@ -0,0 +1,72 @@
+# This workflow is used to update static webpage with documentation for the given GitHub repository.
+# It's triggered by the push of the default branch. 
+# More details: https://docs.google.com/document/d/1WGW4_lOOZ73lEfVZwIWMivbyCgDTJAajxP5N8767DZY
+
+name: pkgdown
+
+on:
+  push:
+    branches: [main, master, test_ci_pkgdown]
+  release:
+    types: [published]
+  workflow_call:
+    inputs:
+      USERNAME:
+        required: true
+        type: string
+      PACKAGE_NAME:
+        required: true
+        type: string
+      BRANCH_NAME:
+        required: true
+        type: string
+      DEFAULT_BRANCH:
```

Comment:
@marcin-kam I would remove it. We define supported branches in l9. What do you think?

Reply:
Sure, I guess it's not needed in this workflow

Reply:
done.